lib/upgrade_html.js

```diff
@@ -392,6 +274,186 @@ function upgradeScriptElement($, docFilename, scriptElem, attrs, hostAttrs,
 }
 
 /**
+ * Upgrades a template element that performs Polymer data binding from 0.5 to
+ * 1.0.
+ *
+ * @param {jQuery} $ The cheerio jQuery object for the document as a whole.
+ * @param {TemplateElement} template The template element to upgrade.
+ * @param {?Array<Object>} opt_newDeclarations An array to put new function
+ *     declarations into. If not given, then it assumes that we're in an auto-
+ *     binding template, and adds HTML comments warning if any expressions
+ *     require functionality moved into a computed expression declaration.
+ */
+function upgradeDataBoundTemplate($, template, opt_newDeclarations) {
+  var newDeclarations = opt_newDeclarations;
```

Comment:
Only in the three or so places that touch newDeclarations, everything else is the same.